src/Auth/DigestAuthenticate.php

```diff
@@ -250,7 +251,7 @@ public function loginHeaders(ServerRequest $request)
     protected function generateNonce()
     {
         $expiryTime = microtime(true) + $this->getConfig('nonceLifetime');
-        $signatureValue = md5($expiryTime . ':' . $this->getConfig('secret'));
+        $signatureValue = hash_hmac('sha256', $expiryTime . ':' . $this->getConfig('secret'), Security::getSalt());
```

Comment:
Don't you have to change the validation of the nonce as well?

Reply:
yes it should, will take care of that

Reply:
There is also a generateNonce method in the test case that is still using md5() as well.

Reply:
updated both